src/components/search/search.tsx

```diff
@@ -48,6 +49,18 @@ const Search: React.FunctionComponent<SearchProps> = ({
         setSearchString(event.target.value);
     };
 
+    const debouncedOnSearch = useMemo(
+        () => debounce(onSearch, 400), // Not sure what this value should be.
```

Comment:
I think 400 is a good starting point.

Reply:
I normally do 800-1000 just bc 400 is too subtle but if you want to keep 400 it's fine.